lib/ui/text/paragraph.h

```diff
@@ -47,12 +54,20 @@ class Paragraph : public ftl::RefCountedThreadSafe<Paragraph>,
 
   static void RegisterNatives(tonic::DartLibraryNatives* natives);
 
+  void toggleTxt();  // Toggle Blink or libtxt backend.
+
  private:
+  friend class ParagraphBuilder;
+
+  std::unique_ptr<ParagraphImpl> m_paragraphImpl;
```

Comment:
Yes, you're holding the ParagraphImpl with a unique_ptr, which means they shouldn't inherit from refcounted or do any of that sort of stuff.